apps/journeys-admin/src/components/DiscoveryJourneys/DiscoveryJourneys.spec.tsx

```diff
@@ -1,15 +1,10 @@
-import { MockedProvider } from '@apollo/client/testing'
 import { render } from '@testing-library/react'
 
 import { DiscoveryJourneys } from '.'
 
 describe('DiscoveryJourneys', () => {
   it('should show embed journeys', () => {
-    const { getByLabelText } = render(
-      <MockedProvider>
-        <DiscoveryJourneys />
-      </MockedProvider>
-    )
+    const { getByLabelText } = render(<DiscoveryJourneys />)
```

Comment:
Just to cover the warning from `DiscoverJourneys.tsx`?

```suggestion
    const { getByLabelText } = render(<DiscoveryJourneys />)
    expect(getByTestId('JourneysAdminDiscoveryJourneys').toBeInTheDocument()
```